src/diamond/convertor.py

```diff
@@ -88,60 +88,12 @@ def convertb(self, value, source, offset=1):
         else:
             source(value * pow(1024, offset))
 
-    def kilobit(self, value=None):
```

Comment:
Yes, they seemed to be private methods. The only method which is being invoked from here is `diamond.convertor.binary.convert`